user-guide/docs/tools/visualization/kalpana.md

```diff
@@ -1,41 +1,102 @@
 ## Kalpana
 <a name="kaplana-user-guide"></a><!-- old heading name/id -->
 
-!!! note
-	Kalpana is **temporarily unavailable**. It is being updated.
+Kalpana is a Python library which primarily converts ADCIRC NetCDF output files to GIS-compatible shapefiles. Secondary functions are not available in the DesignSafe application. It was developed by the Coastal and Computational Hydraulics Team at North Carolina State University. More information regarding Kalpana can be found at:
```

Comment:
1. Consider removing "regarding Kalpana", because user is reading doc for Kaplana, so the context seems obvious.
2. Consider removing "can be found at", because it does not seem to assist the reader. The phrase ends with a colon, so a complete sentence is not necessary.

```suggestion
Kalpana is a Python library which primarily converts ADCIRC NetCDF output files to GIS-compatible shapefiles. Secondary functions are not available in the DesignSafe application. It was developed by the Coastal and Computational Hydraulics Team at North Carolina State University. More information:
```